android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/adding/AddingFragment.java

```diff
@@ -0,0 +1,240 @@
+package de.dhbw.ka.se.fibo.ui.adding;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.NavController;
+import androidx.navigation.Navigation;
+
+import com.google.android.material.button.MaterialButton;
+import com.google.android.material.datepicker.MaterialDatePicker;
+import com.google.android.material.tabs.TabLayout;
+import com.google.android.material.textfield.MaterialAutoCompleteTextView;
+import com.google.android.material.textfield.TextInputEditText;
+
+import java.math.BigDecimal;
+import java.text.Format;
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import de.dhbw.ka.se.fibo.ApplicationState;
+import de.dhbw.ka.se.fibo.R;
+import de.dhbw.ka.se.fibo.databinding.FragmentAddingBinding;
+import de.dhbw.ka.se.fibo.models.Cashflow;
+import de.dhbw.ka.se.fibo.models.CashflowType;
+import de.dhbw.ka.se.fibo.models.Category;
+import de.dhbw.ka.se.fibo.models.Place;
+
+public class AddingFragment extends Fragment {
+
+    private FragmentAddingBinding binding;
+    private MaterialDatePicker<Long> datePicker;
+    private NavController navController;
+    private TextInputEditText store;
+    private TextInputEditText amount;
+    private TextInputEditText dateText;
+    private MaterialAutoCompleteTextView categoriesDropdown;
+    private MaterialButton cancelButton;
+    private MaterialButton okayButton;
```

Comment:
maybe we call it safeButton, but I also agree with okayButton

Reply:
Should the button also have the text "save" instead of "okay"?